pkg/controller/controller.go

```diff
@@ -174,12 +179,23 @@ func (c *CSIControllerService) CreateVolume(ctx context.Context, req *csi.Create
 		ctxValue = context.WithValue(ctx, util.VolumeInfoKey, volumeInfo)
 	)
 
-	if accessType, ok := req.GetVolumeCapabilities()[0].AccessType.(*csi.VolumeCapability_Mount); ok {
-		fsType = strings.ToLower(accessType.Mount.FsType) // ext4 by default (from request)
-		mode = apiV1.ModeFS
-	} else {
-		mode = apiV1.ModeRAW
+	accessType, ok := req.GetVolumeCapabilities()[0].AccessType.(*csi.VolumeCapability_Mount)
```

Comment:
This construction is unclear for me

- why additional block needed?
- switch for just 2 case is a smell code

This construction is might lead to `panic` - check slice before accessing elements. `req.GetVolumeCapabilities()[0]`
 
It's better to declare default at first and their other cases, for example%

```go
// Comments goes here
mode = apiV1.ModeRAW
if accessType, ok := req.GetVolumeCapabilities()[0].AccessType.(*csi.VolumeCapability_Mount); ok {
  // Comments goes here
  // ext4 by default (from request)
  fsType = strings.ToLower(accessType.Mount.FsType)
  mode = apiV1.ModeFS
}
if isNeedRawPart(req.GetParameters()) {
  // Comments goes here
  mode = apiV1.ModeRAWPART
}
// The rest of the code
```

Reply:
Fixes:
- Add check, if VolumeCapabilities are empty
- Refactor switch with ifelse
- Add comments

Reply:
What do you mean by ` additional block needed`?

Reply:
```
switch foo {
case true:
  { <- this is additional block, what the purpose of this block ?
    // 
  }
}
```

Reply:
Got it, thank you!